lms/urls.py

```diff
@@ -33,6 +33,11 @@
     url(r'', include('student.urls')),
     # TODO: Move lms specific student views out of common code
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
+
+    url(r'^course/{}/refund_status$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
Refunds should be handled in Otto. This work literally has nothing to do with courses, so using the `/course` path makes no sense.